app/models/Item.scala

```diff
@@ -319,7 +319,7 @@ object Item {
 
   def deleteBefore(date: Date) {
     DB.withConnection { implicit c =>
-      val rows = SQL("select id from item where created < {created}").on('created -> date)
+      val rows = SQL("select id from item where created <= {created}").on('created -> date)
```

Comment:
This is one of the few functional changes in this series of commits. I believe it reflects the likely intent of the original code, but if not I can change it back and update the test to reflect that the date value being passed in will not actually be deleted.